xAPI.md

```diff
@@ -2643,23 +2643,52 @@ the owner of that extension IRI explicitly states that a particular form of code
 
 ## 1.0 Requests
 
-### 1.1 Request Types
+xAPI tracking is done via HTTP Requests from the Activity Provider (client) to the LRS (server).  This 
+specification offers guidance in some aspects of this communication.  Where no guidance is offered, it is 
+recommended that those implementing xAPI use current industry best practices.
+
+### 1.1 Request Methods
+
+xAPI supports the use of PUT, POST, GET, DELETE, and HEAD.  Use of other methods is beyond the scope of xAPI.  Each of the following sections details the use of each method and how xAPI may differ slightly.  For full 
+implementation details, see each of the [Resources](#resources) below.
 
 #### 1.1.1 HTTP PUT
 
+The HTTP PUT method is used to either create or update.  The enclosed entity is stored by the corresponding  
+Resource.  If it is new, it is created.  If there is a conflict, it is updated.  xAPI has a major exception to this 
+in that Statements are not allowed to be updated and thus there are rules around using PUT to update them.  Also in 
+xAPI, a PUT can only add a single Statement at a time.
+
+A major difference between PUT and POST is that PUT is always idempotent, meaning if the method is executed more than once, the same effect occurs as if it were executed just one time.  POST may require additional information 
+to be idempotent. 
+
 #### 1.1.2 HTTP POST
 
+Like PUT, the enclosed entity is stored by the corresponding Resource.  However, POST methods to any of the 
```

Comment:
Here and above we use "enclosed", but HTTP-transmitted entities aren't enclosed, which is used to talk about wrapped payloads in approaches such as SOAP (or in our POST-burrowed option). When thinking more about the sentence, "stored by the corresponding Resource" doesn't really make sense either -- it has the right sort of words, but it doesn't align with their correct usages. Resources don't store things, resources *are* things. Also, in the case of PUT you're addressing the a resource's location directly, but in the case of POST you often address the resource representing all of the things of a type, such as in the Statement API (but not when used with our document APIs).